themes-default/slim/views/inc_qualityChooser.mako

```diff
@@ -1,5 +1,54 @@
+<script type="text/x-template" id="quality-chooser-template">
```

Comment:
This file needs to be in themes-default/slim/views/vue-components/qualityChooser.mako

Reply:
Did you want me to replace this file with the old content and just move the new code to a new file?